mainloop.lua

```diff
@@ -385,9 +385,11 @@ function Stack.wait_for_random_character(self)
 end
 
 function Stack.handle_pause(self)
-  if menu_pause() then
+  if menu_pause() or not love.window.hasFocus() then
```

Comment:
```suggestion
  if menu_pause() or (not GAME.focused and not GAME.gameIsPaused) then
```

Then remove the inner if. This just cleans it up a bit and makes it so we don't run the code below over and over again while not focused.

Reply:
Done.